sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/Distribute.java

```diff
@@ -1119,6 +1127,39 @@ private void viewReleaseNotes(ReleaseDetails releaseDetails) {
     }
 
     /**
+     * Store failed package info hash in preferences.
+     */
+    private synchronized void storeUpdateSetupFailedPackageHash() {
```

Comment:
pass dialog as parameter and check against that instead of null

Reply:
Got it, thanks for pointing